pkg/virt-controller/services/template.go

```diff
@@ -344,13 +344,90 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 	})
 
 	volumeMounts = append(volumeMounts, k8sv1.VolumeMount{
-		Name:      "virt-share-dir",
-		MountPath: t.virtShareDir,
+		Name:      "libvirt-runtime",
+		MountPath: "/var/run/libvirt",
 	})
 
+	// virt-handler domain notify socket
 	volumeMounts = append(volumeMounts, k8sv1.VolumeMount{
-		Name:      "libvirt-runtime",
-		MountPath: "/var/run/libvirt",
+		Name:      "virt-handler-domain-notify",
+		MountPath: filepath.Join(t.virtShareDir, "domain-notify.sock"),
+	})
+	volumes = append(volumes, k8sv1.Volume{
+		Name: "virt-handler-domain-notify",
+		VolumeSource: k8sv1.VolumeSource{
+			HostPath: &k8sv1.HostPathVolumeSource{
+				Path: filepath.Join(t.virtShareDir, "domain-notify.sock"),
+			},
+		},
+	})
+
+	// selinux cil files
+	volumeMounts = append(volumeMounts, k8sv1.VolumeMount{
+		Name:      "virt-launcher-cil",
+		MountPath: filepath.Join(t.virtShareDir, "virt_launcher.cil"),
+	})
+	volumes = append(volumes, k8sv1.Volume{
+		Name: "virt-launcher-cil",
+		VolumeSource: k8sv1.VolumeSource{
+			HostPath: &k8sv1.HostPathVolumeSource{
+				Path: filepath.Join(t.virtShareDir, "virt_launcher.cil"),
+			},
+		},
+	})
+	volumeMounts = append(volumeMounts, k8sv1.VolumeMount{
+		Name:      "base-container-cil",
+		MountPath: filepath.Join(t.virtShareDir, "base_container.cil"),
+	})
+	volumes = append(volumes, k8sv1.Volume{
+		Name: "base-container-cil",
+		VolumeSource: k8sv1.VolumeSource{
+			HostPath: &k8sv1.HostPathVolumeSource{
+				Path: filepath.Join(t.virtShareDir, "base_container.cil"),
+			},
+		},
+	})
```

Comment:
oh, I think these are needed for virt-handler to install it, not for virt-launcher

Reply:
ah, i see, so it isn't relevant to share the .cil files with virt-launcher. thanks!